views/peeps.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<h1> Welcome to Chitter!</h1>
+
+<h3>Check out the Chitter feed below:</h3>
+<ul>
+  <% @peeps.reverse_each do |peep| %>
```

Comment:
Wow I never knew about reverse_each! This seems a lot simpler than what I did lol